### PR TITLE
Login help styling

### PR DESCRIPTION
**Pull request by Ambradibi**
https://github.com/aegisbigdata/documentation/issues/145

### CODE REVIEW

File: .directory [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```
Comment (cbotsikas) [inline .directory]: What is this file? maybe you need to add it to .gitignore

File: hopsworks-admin/src/main/java/io/hops/hopsworks/admin/user/account/ResetPassword.java [hunk 2]
```diff
@@ -503,7 +503,7 @@ public String logout() {
       req.logout();
       session.invalidate();
       usersController.setOnline(people.getUid(), -1);
-      context.getExternalContext().redirect("/hopsworks/#!/home");
+      context.getExternalContext().redirect("/aegis-web/#!/home");
```
Comment (cbotsikas) [inline hopsworks-admin/src/main/java/io/hops/hopsworks/admin/user/account/ResetPassword.java]: shouldn't this be `/aegis/#!/home`? what is aegis-web?

File: hopsworks-admin/src/main/webapp/WEB-INF/faces-config.xml [hunk 3]
```diff
@@ -250,6 +250,11 @@
       <to-view-id>/login.xhtml</to-view-id>
       <redirect/>
     </navigation-case>
+    <navigation-case>
+      <from-outcome>welcome-aegis</from-outcome>
+      <to-view-id>/aegis-web/#!/login</to-view-id>
```
Comment (cbotsikas) [inline hopsworks-admin/src/main/webapp/WEB-INF/faces-config.xml]: shouldn't this be `/aegis/#!/login`?

File: hopsworks-admin/src/main/webapp/security/protected/admin/admintitleBar.xhtml [hunk 4]
```diff
@@ -50,7 +50,8 @@
       <h:form id="toolbarForm"> 
         <p:toolbar>
           <f:facet name="left" rendered="#{roleEnforcementPoint.loggedIn}">
-            <p:commandButton value="Hopsworks" action="#{facesContext.externalContext.redirect('/hopsworks')}" ajax="false" process="@toolbarForm" icon="ui-icon-extlink"/>
+            <p:commandButton value="Aegis" action="#{facesContext.externalContext.redirect('/aegis-web')}" ajax="false" process="@toolbarForm" icon="ui-icon-extlink"/>
```
Comment (cbotsikas) [inline hopsworks-admin/src/main/webapp/security/protected/admin/admintitleBar.xhtml]: shouldn't this be `/aegis`?

File: hopsworks-admin/src/main/webapp/security/protected/admin/admintitleBar.xhtml [hunk 5]
```diff
@@ -50,7 +50,8 @@
...
+            <!--<p:commandButton value="Hopsworks" action="#{facesContext.externalContext.redirect('/aegis-web')}" ajax="false" process="@toolbarForm" icon="ui-icon-extlink"/>-->
```
Comment (cbotsikas) [inline hopsworks-admin/src/main/webapp/security/protected/admin/admintitleBar.xhtml]: same here but since it's a comment could be deleted.

**Review by cbotsikas — CHANGES_REQUESTED** [review]

**Comment by Ambradibi** [comment]
@cbotsikas I didn't change these files. I ask colleagues for the answer to your questions

File: hopsworks-admin/src/main/java/io/hops/hopsworks/admin/user/account/ResetPassword.java (hunk 2 above)
Comment (Ambradibi) [reply hopsworks-admin/src/main/java/io/hops/hopsworks/admin/user/account/ResetPassword.java]: changed

File: hopsworks-admin/src/main/webapp/WEB-INF/faces-config.xml (hunk 3 above)
Comment (Ambradibi) [reply hopsworks-admin/src/main/webapp/WEB-INF/faces-config.xml]: changed

File: hopsworks-admin/src/main/webapp/security/protected/admin/admintitleBar.xhtml (hunk 5 above)
Comment (Ambradibi) [reply hopsworks-admin/src/main/webapp/security/protected/admin/admintitleBar.xhtml]: done

File: hopsworks-admin/src/main/webapp/security/protected/admin/admintitleBar.xhtml (hunk 4 above)
Comment (Ambradibi) [reply hopsworks-admin/src/main/webapp/security/protected/admin/admintitleBar.xhtml]: changed

File: .directory (hunk 1 above)
Comment (Ambradibi) [reply .directory]: done

**Review by cbotsikas — APPROVED** [review]
Code looks ok. Test is pending to proceed with merge.

**Comment by cbotsikas** [comment]
fixes https://github.com/aegisbigdata/documentation/issues/146 too

**Pull request merged**